libbeat/outputs/logstash/client.go

```diff
@@ -71,6 +72,16 @@ func newLumberjackClient(
 	}
 }
 
+func (l *lumberjackClient) Connect(timeout time.Duration) error {
```

Comment:
Connect and Close are used by output/modes module. Just adding some debug here to better see how window size changes and reconnects/errors happen (as these are related). We re-connect on failure + we adapt window sizes on error.